lib/matplotlib/_constrained_layout.py

```diff
@@ -87,14 +87,18 @@ def do_constrained_layout(fig, h_pad, w_pad,
        of 0.1 of the figure width between each column.
        If h/wspace < h/w_pad, then the pads are used instead.
 
+    rect : tuple of 4 floats
+        Rectangle in figure coordinates to perform constrained layout in
+        [left, bottom, width, height], each from 0-1.
```

Comment:
```suggestion
        (left, bottom, width, height), each from 0-1.
```

Reply:
Should one possibly add default values here? (And in other locations.)

Reply:
Btw, doesn't look like there is a renderer parameter (anymore?).

Reply:
Yes, needed to rebase.